Legendary Rune Maker/Pages/MainPage.xaml.cs

```diff
@@ -312,6 +312,7 @@ private void Load_Click(object sender, EventArgs e)
 
                                 LogTo.Debug("Setting page tree");
                                 Tree.SetPage(page);
+                                SelectedPosition = page.Position;
```

Comment:
```suggestion
```

I don't like this because I want to let the user load runes from any position if they find that better.